src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let memory = [];
+
+  const adder = (number) => {
+    if (number === undefined) {
```

Comment:
Using strict comparison with 'undefined' is fine, but it's more idiomatic to check if a parameter is provided using `typeof number === 'undefined'` or simply `number === undefined` without the strict comparison.